config/initializers/security_headers.rb

```diff
@@ -1,13 +1,13 @@
-::SecureHeaders::Configuration.configure do |config|
-  config.hsts                   = {:max_age => 99, :include_subdomains => true}
+::SecureHeaders::Configuration.default do |config|
+  config.hsts                   = 'max_age=99; include_subdomains=true'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.